megamek/src/megamek/common/pathfinder/ShortestPathFinder.java

```diff
@@ -370,13 +370,9 @@ public static int getFacingDiff(final MovePath mp, Coords dest,
      * @param board
      * @return
```

Comment:
```suggestion
     * @param ignore
     * @return
```
Also needs descriptive text for the parameters